### ISSUE TITLE
Check: CKV_OCI_21 Ensure security group has stateless ingress security rules, incorrect logic

### ISSUE THREAD

**Issue by patrijua** [bridgecrewio/checkov#6745]
**Describe the issue**
`CKV_OCI_21` rule documentation recommends that ingress security rules should be stateful, so `stateless` should be set to false. Also the documentation seems to be not consistent so there is mixed up information on it.
 
https://docs.prismacloud.io/en/enterprise-edition/policy-reference/oci-policies/networking/bc-oci-21

When `oci_core_network_security_group_security_rule` -resource has argument `stateless = false` the rule reports it as failed. I believe stateful is the desired value so rule should not report this.

**Examples**
```
resource "oci_core_network_security_group_security_rule" "example_ingress" {
  count = length(local.example_ingress)

  network_security_group_id = oci_core_network_security_group.example.id
  direction                 = "INGRESS"
  source                    = local.example_ingress[count.index].source
  source_type               = local.example_ingress[count.index].source_type
  protocol                  = local.example_ingress[count.index].protocol
  stateless                 = false
  description               = local.example_ingress[count.index].description
  ...
```
```
Check: CKV_OCI_21: "Ensure security group has stateless ingress security rules"
	FAILED for resource: oci_core_network_security_group_security_rule.example_ingress
```

**Version (please complete the following information):**
 - Checkov Version 3.2.250

**Additional context**

**Comment by itariq20** [comment]
@patrijua Hi, thank you for pointing the issue out and providing the relevant details, we are investigating this issue internally and we’ll update you with our findings.

**Comment by tsmithv11** [comment]
Hey @patrijua thanks for reaching out. In this case, the logic was accurate but the docs were wrong. The PR above fixes that and should be more clear.
